core/domain/issue_registry_test.py

```diff
@@ -0,0 +1,38 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for methods in the issue registry."""
+
+from core.domain import issue_registry
+from core.tests import test_utils
+
+
+class BaseExplorationIssueSpec(object):
```

Comment:
@pranavsid98 are we planning to delete this? If so, please file an issue for it and link to it in this comment thread.

Thanks!